### ISSUE TITLE
How do I use these in a project with CocoaPods

### ISSUE THREAD

**Issue by edasque** [pNre/ExSwift#24]
I've created the Podfile and installed ExSwift. I see it in my xcworkspace.

It's probably not enough but I can't figure out what to do next.

![anagram_xcodeproj](https://cloud.githubusercontent.com/assets/219187/3729478/3047b356-16be-11e4-8620-a4ef8e066d1b.png)

**Comment by edasque** [comment]
Still a problem?

**Comment by pNre** [comment]
Yes, nothing changed. I don't think we'll se any change until the next Xcode version (6.3 - 7).

**Comment by truppelito** [comment]
@pNre Has the new Xcode beta (6.3) fixed this problem? I'd looove if this library was Carthage compatible, but if it still can't be compiled as a framework then it isn't even possible, let alone compatible.

**Comment by pNre** [comment]
I did a quick test with CocoaPods and it seems to be working!
